helm/raster/templates/configmap.yaml

```diff
@@ -34,7 +34,7 @@ data:
   AUTH_ENABLED: {{ .Values.nginx.authorization.domain | quote }}
   {{ if and (not .Values.nginx.opalaEnabled) (.Values.nginx.authorization.domain) }}
```

Comment:
there is no such value `.Values.nginx.opalaEnabled`
is `not` needs to be removed?